src/app/application-dialog/application-dialog.component.ts

```diff
@@ -88,30 +88,32 @@ export class ApplicationDialogComponent implements OnInit {
       ],
       monthlyIncome: [
         this.applicationData.monthlyIncome as number, {
-          validators: [Validators.required, Validators.pattern('[0-9]*'),Validators.max(4999999999)],
+          validators: [Validators.required, Validators.pattern('[0-9]*'), Validators.max(4999999999)],
```

Comment:
4999999999 should be const in top somewhere, would be pain when refactoring all values